specs/commandline/plugin.md

```diff
@@ -41,51 +41,115 @@ Aliases:
-  -h, --help                    help for install
-  -f, --force                   force the installation of a plugin
+  -d, --debug              debug mode
+      --file               install plugin from a file in file system
+      --force              force the installation of a plugin
+  -h, --help               help for install
+      --sha256sum string   must match SHA256 of the plugin source
+      --url                install plugin from an HTTPS URL
+  -v, --verbose            verbose mode
 
 Aliases:
   install, add
 ```
 
-### notation plugin remove
+### notation plugin uninstall
 
 ```text
-Removes a plugin
+Uninstall a plugin
 
 Usage:
-  notation plugin remove [flags] <plugin name>
+  notation plugin uninstall [flags] <plugin_name>
 
 Flags:
   -h, --help          help for remove
-
+  -y, --yes           do not prompt for confirmation
 Aliases:
-  remove, rm, uninstall, delete
+  uninstall, remove, rm
 ```
 
 ## Usage
 
-### Install a plugin
+## Install a plugin 
+
+### Install a plugin from file system
+
+Install a Notation plugin from file system. Plugin file supports `.zip` and `.tar.gz` format. The checksum validation is optional for this case. 
 
 ```shell
-notation plugin install <plugin path>
+$ notation plugin install --file <file_path>
+```
+
+Upon successful execution, the plugin is copied to Notation's plugin directory. If the plugin directory does not exist, it will be created. The name and version of the installed plugin are displayed as follows. 
+
+```console
+Successfully installed plugin <plugin name>, version <x.y.z>
+```
+
+If the entered plugin checksum digest doesn't match the published checksum, Notation will return an error message and will not start installation.
+
+```console
+Error: failed to install the plugin: plugin checksum does not match user input. Expecting <sha256sum>
+```
+
+If the plugin version is higher than the existing plugin, Notation will start installation and overwrite the existing plugin.
+
+```console
+Successfully installed plugin <plugin name>, updated the version from <x.y.z> to <a.b.c>
+```
+
+If the plugin version is equal to the existing plugin, Notation will not start installation and return the following message. Users can use a flag `--force` to skip plugin version check and force the installation.
+
+```console
+Plugin <plugin name> with version <x.y.z> already exists. 
+To view a list of installed plugins, use "notation plugin list"
+```
+
+If the plugin version is lower than the existing plugin, Notation will return an error message and will not start installation. Users can use a flag `--force` to skip plugin version check and force the installation.
+
+```console
+Error: failed to install the plugin: <plugin-name>. The installing version <a.b.c> is lower than the existing plugin version <x.y.z>.
+It is not recommended to install an older version. Use "--force" to force the installation if you want to do so. To view a list of installed plugins, use "notation plugin list"
```

Comment:
```suggestion
Unable to install the plugin: <plugin-name>. The version you are trying to install (<a.b.c>) is older than the version that is already installed (<x.y.z>). To force the installation, use the "--force" option.
```

Reply:
@priteshbandi  To align the behavior with the case when installing version is equal to exist plugin as you described in https://github.com/notaryproject/notation/pull/809#discussion_r1396788077, I think this output should be stderr either.

Reply:
I concur, updated messsage
```
Error: Unable to install the plugin: <plugin-name>. The version you are trying to install (<a.b.c>) is equal or lower than the version that is already installed (<x.y.z>). To force the installation, use the "--force" option.
```